igvc_navigation/launch/back_circle_server.launch

```diff
@@ -0,0 +1,9 @@
+<launch>
+    <node name="back_circle" pkg="igvc_navigation" type="back_circle" output="screen" required="true">
+        <param name="width" value="1"/>
+        <param name="length" value="1.4"/>
+        <param name="grid_size" value="0.1"/>
+        <param name="thickness" value="0.2"/>
+        <param name="offset" value="-0.5"/>
```

Comment:
Could we set these parameters in a yaml file like `global_costmap_params.yaml` instead of here?

Reply:
I'll make it so the config class handles this.

Reply:
Actually because the global_costmap_params.yaml file controls the layers and not the BackCircleServer, it can't be done.